test/runtests.py

```diff
@@ -99,9 +103,19 @@ def test_path(folder, is_file):
                 baseline = os.path.splitext(js_file)[0] + '.baseline'
                 baseline = os.path.join(folder, baseline)
                 if os.path.isfile(baseline):
-                    expected_output = None
+                    baseline_output = None
                     with open(baseline, 'r') as bs_file:
-                        expected_output = bs_file.read().replace('\r', '')
+                        baseline_output = bs_file.read()
+
+                    # clear carriage return
+                    # todo: remove carriage return at the end of the line
+                    #       or better fix ch to output same on all platforms
+                    expected_output = baseline_output.replace('\r', '')
+
+                    # normalize tabs on both outputs to 2 spaces
+                    expected_output = expected_output.replace('\t', '  ')
```

Comment:
I didn't get why you need to process `\t`. It doesn't matter what is displayed. If both baseline file and captured output use `\t`, they should match.

If you replaced them here and `show_failed` prints to log file, people will actually be confused when comparing with baseline files, because now the log has `'  '` while baseline has `\t`.